lib/tasks/jobs.rake

```diff
@@ -0,0 +1,7 @@
+namespace :jobs do
+  desc 'Populate published_at field'
+  task populate_job_field: :environment do
+    Job.where(published_at: nil, state: :published)
```

Comment:
Happy you also like this approach. I'm prefer to have the data manipulation method inside rake task instead of migration. I'm always nervous with DB 😛